src/components/parts/alert-panel/AlertPanelPart.tsx

```diff
@@ -30,7 +30,7 @@ export const AlertPanelPart = ({ config }: AlertPanelPartProps) => {
                 </LenkeBase>
             )}
         >
-            <Information className={style.informationIcon} />
+            <InformationSquareIcon className={style.informationIcon} />
```

Comment:
Er det riktig at dette ikke skjules for skjermleser?

Reply:
Nei. Takk.